helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/steps/TicketSteps.java

```diff
@@ -157,6 +159,19 @@ public void verifyIdleMessage() throws StreamsException, MessagesException, Inte
 
   }
 
+  @Then("bot can verify there is no message created in the queue room")
+  public void verifyNoneMessage() throws StreamsException, MessagesException, InterruptedException {
+
+    Optional<SymMessage> message = messageHelper.getLatestQueueRoomMessage(initialTime);
+
+    Optional<Ticket> ticket = ticketHelper.getUnservicedTicket();
+
+    String expectedString = String.format(IDLE_TICKET_MESSAGE, ticket.get().getId());
```

Comment:
The expected message should be the question message rather than the idle message.